cnv/cnv_users_guide/cnv-using-the-default-pod-network-with-cnv.adoc

```diff
@@ -0,0 +1,31 @@
+[id="cnv-using-the-default-pod-network-with-cnv"]
+= Using the default Pod network with {ProductName}
+include::modules/cnv-document-attributes.adoc[]
+:context: cnv-using-the-default-pod-network-with-cnv
+toc::[]
+
+You can use the default Pod network with {ProductName}. To do so,
+you must use the `masquerade` binding method. It is the only recommended
+binding method for use with the default Pod network. Do not use
+`masquerade` mode with non-default networks.
+
+[NOTE]
+====
+For secondary networks, use the `bridge` binding method.
```

Comment:
Does this need to be in a note?

Reply:
Probably not, but I wanted to make a clear distinction. Do you think it's alright as-is?

Reply:
It's fine as is. Previously I've heard people complain that we use too many note blocks, and it can devalue each one; however, I don't think this requires a change.